webhook.go

```diff
@@ -222,218 +228,236 @@ func hookHandler(w http.ResponseWriter, r *http.Request) {
 
 	log.Printf("[%s] incoming HTTP request from %s\n", rid, r.RemoteAddr)
 
+	id := mux.Vars(r)["id"]
+
+	matchedHook := matchLoadedHook(id)
+	if matchedHook == nil {
+		w.WriteHeader(http.StatusNotFound)
+		fmt.Fprint(w, "Hook not found.")
+		return
+	}
+
+	// Check for allowed methods
+	if len(matchedHook.HTTPMethods) != 0 {
+		var allowed bool
+		for i := range matchedHook.HTTPMethods {
+			if matchedHook.HTTPMethods[i] == r.Method {
```

Comment:
We should normalize strings to uppercase, either here using `strings.ToUpper`, or during the hook load, so that hooks using "get", "GeT", etc... in their hook "http-methods", also get matched.

Reply:
Doing it per request now.  We'll need to refactor config loading in a separate PR.